src/utils/fetchCurrentUser.js

```diff
@@ -0,0 +1,9 @@
+export default function fetchCurrentUser() {
+
+    return {
+        "username": "grumpy19",
```

Comment:
This is a very creative workaround, well done :) we could also hardcode this or use Context